Source/Core/InputCommon/InputCommon.vcxproj

```diff
@@ -36,6 +36,9 @@
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
   <ItemGroup>
+    <ClCompile Include="ControllerEmu\ControlGroup\IMUAccelerometer.cpp" />
```

Comment:
These new files will need to be added to the CMakeLists.txt.

Reply:
I also added these:
```
ControllerInterface/CemuHookUDPServer/CemuHookUDPServer.cpp
ControllerInterface/CemuHookUDPServer/CemuHookUDPServer.h
ControllerInterface/CemuHookUDPServer/CemuHookUDPServerProto.h
```